zingo-cli/src/lib.rs

```diff
@@ -377,9 +415,15 @@ pub type CommandRequest = (String, Vec<String>);
 /// Command responses are strings
 pub type CommandResponse = String;
 
-/// Used by the zingocli crate, and the zingo-mobile application:
-/// <https://github.com/zingolabs/zingolib/tree/dev/cli>
-/// <https://github.com/zingolabs/zingo-mobile>
+/// Builds a `ZingoConfig` and initializes a `LightClient` from (in order):
+/// an explicit seed phrase, a UFVK, an existing wallet file, or fresh entropy.
+///
+/// For fresh wallets, the birthday is chosen as follows: if an indexer is configured,
+/// we query the chain tip and start at `tip - 100`. Otherwise we
+/// fall back to a conservative activation height and create the wallet offline.
```

Comment:
```suggestion
/// For fresh wallets, the birthday is chosen as follows: if an indexer is configured,
/// we query the chain height and start at `chain height - 100`. Otherwise we
/// fall back to a conservative activation height and create the wallet offline.
```

to stay in line with pepper-sync terminology